### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by MuskanChhatrasal**

### CODE REVIEW

**Review by aayushjain-code — COMMENTED** [review]
try to fix navbar is not aligned properly

**Comment by sainath7878** [comment]
Home and WOW SHOP are overlapping

File: components/Navbar.css [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+  margin-top: -1rem;
+}
+
+header .logo {
+  font-size: 2rem;
+  font-weight: 600;
+  color: var(--primary-dark-color);
+  margin-top: -3.5rem;
+  margin-left: 1.8rem;
+  /* margin: 0.1rem; */
```
Comment (sainath7878) [inline components/Navbar.css]: Remove commented code

File: index.html [hunk 2]
```diff
@@ -4,10 +4,47 @@
...
+        integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
     <link rel="stylesheet" href="styles.css">
+    <title>WOW SHOP</title>
 </head>
 <body>
+    <header>
+        <div id="menu-bar" class="fas fa-bars"></div>
+    
+        <div class="left-container">
+           <img src="./assets/Cartoon-chef-man-with-thumb-finger-on-transparent-background-PNG-removebg-preview.png" class='logo-img' alt="" srcset="">
```
Comment (sainath7878) [inline index.html]: alt is left empty

File: components/Navbar.css [hunk 3]
```diff
@@ -0,0 +1,216 @@
+@import url("https://fonts.googleapis.com/css2?family=Lobster&family=Permanent+Marker&family=Roboto:ital,wght@0,300;0,400;0,500;0,700;0,900;1,300;1,500;1,900&display=swap");
+header {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  z-index: 1000;
+  background: #fff;
```
Comment (sainath7878) [inline components/Navbar.css]: use variables for colors.

**Comment by MuskanChhatrasal** [comment]
Okayy!! Will  do that.
Thankyou.

**Comment by MuskanChhatrasal** [comment]
Okayy!! Working on it.
Thankyou.

File: components/Navbar.css [hunk 4]
```diff
@@ -0,0 +1,216 @@
...
+}
+
+header .navbar {
+  margin-left: -32rem;
+  margin-top: 2rem;
+}
+header .navbar a {
+  font-size: 1rem;
+  font-weight: 600;
+  color: var(--black);
```
Comment (sainath7878) [inline components/Navbar.css]: It's not recommended to use color name as variable name.

File: components/Navbar.css (hunk 1 above)
Comment (MuskanChhatrasal) [reply components/Navbar.css]: Okayy!! Will do that.

File: components/Navbar.css [hunk 5]
```diff
@@ -0,0 +1,216 @@
...
+  text-align: center;
+  pointer-events: none;
+  color: #aaa;
+}
+
+.serach-box {
+  width: 20rem;
+}
+
+/* .button {
```
Comment (sainath7878) [inline components/Navbar.css]: This also can be removed.

File: components/Navbar.css [hunk 6]
```diff
@@ -0,0 +1,216 @@
...
+  #menu-bar {
+    display: block;
+  }
+
+  header .navbar {
+    position: absolute;
+    top: 100%;
+    left: -120%;
+    width: 100%;
+    background: #eee;
```
Comment (sainath7878) [inline components/Navbar.css:238]: Variable name can be used

Comment (MuskanChhatrasal) [reply components/Navbar.css:238]: Okayy!! 
Thankyou

File: components/Navbar.css (hunk 5 above)
Comment (MuskanChhatrasal) [reply components/Navbar.css]: This I have removed. You can see the updated one

**Pull request merged**